test/tc_basic.rb

```diff
@@ -114,7 +114,7 @@ def test_set_and_get_point_from_wkt
               obj_ = klass_.new
               assert_nil(obj_.latlon)
               obj_.latlon = 'SRID=1000;POINT(1 2)'
-              assert_equal(@factory.point(1, 2), obj_.latlon)
+              assert_equal(RGeo::Cartesian.preferred_factory(srid: 1000).point(1, 2), obj_.latlon)
```

Comment:
@januszm here's my Gemfile.lock:

```
GIT
  remote: https://github.com/rgeo/rgeo-activerecord.git
  revision: 0d3f7975b476135b7baf8b1831b4fa2528500616
  branch: 1.0
  specs:
    rgeo-activerecord (1.2.0)
      activerecord (~> 4.0)
      rgeo (~> 0.3)

PATH
  remote: .
  specs:
    activerecord-mysql2spatial-adapter (0.5.0)
      activerecord (>= 4.0, < 4.2)
      mysql2 (>= 0.2.13, < 0.4.0)
      rgeo-activerecord (~> 1.0)

GEM
  remote: https://rubygems.org/
  specs:
    activemodel (4.1.16)
      activesupport (= 4.1.16)
      builder (~> 3.1)
    activerecord (4.1.16)
      activemodel (= 4.1.16)
      activesupport (= 4.1.16)
      arel (~> 5.0.0)
    activesupport (4.1.16)
      i18n (~> 0.6, >= 0.6.9)
      json (~> 1.7, >= 1.7.7)
      minitest (~> 5.1)
      thread_safe (~> 0.1)
      tzinfo (~> 1.1)
    appraisal (2.2.0)
      bundler
      rake
      thor (>= 0.14.0)
    arel (5.0.1.20140414130214)
    builder (3.2.3)
    coderay (1.1.1)
    i18n (0.8.4)
    json (1.8.6)
    method_source (0.8.2)
    minitest (5.10.2)
    mysql2 (0.3.21)
    pry (0.10.4)
      coderay (~> 1.1.0)
      method_source (~> 0.8.1)
      slop (~> 3.4)
    rake (12.0.0)
    rdoc (5.1.0)
    rgeo (0.6.0)
    slop (3.6.0)
    thor (0.19.4)
    thread_safe (0.3.6)
    tzinfo (1.2.3)
      thread_safe (~> 0.1)

PLATFORMS
  ruby

DEPENDENCIES
  activerecord-mysql2spatial-adapter!
  appraisal
  pry
  rake (>= 0.9.2)
  rdoc (>= 3.12)
  rgeo-activerecord!

BUNDLED WITH
   1.15.1
```

Also, I installed GEOS to see if it would fix any of the failures; it did not.